infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java

```diff
@@ -98,7 +100,7 @@ public static byte[] tryCompress(String str, String encoding) {
         try (GZIPOutputStream gzip = new GZIPOutputStream(out)) {
             gzip.write(str.getBytes(encoding));
         } catch (Exception e) {
-            e.printStackTrace();
+            log.info("gzip write is fail:{}", JSONUtil.isJson(e.toString()));
```

Comment:
Why do you need json processing of exception stack?

Reply:
write casually. ok， i will update and commit!

Reply:
should print error level logs

Reply:
ok